stories/modules-tagmanager-setup.stories.js

```diff
@@ -49,7 +49,51 @@ import {
 import { MODULES_ANALYTICS } from '../assets/js/modules/analytics/datastore/constants';
 import * as fixtures from '../assets/js/modules/tagmanager/datastore/__fixtures__';
 import * as modulesFixtures from '../assets/js/googlesitekit/modules/datastore/__fixtures__';
-import { parseLiveContainerVersionIDs } from '../assets/js/modules/tagmanager/datastore/__factories__/utils';
+import {
+	createBuildAndReceivers,
+	parseLiveContainerVersionIDs,
+} from '../assets/js/modules/tagmanager/datastore/__factories__/utils';
+
+function buildAndReceiveProperties( registry, accountID, internalContainerID ) {
+	const propertyID = registry
+		.select( MODULES_TAGMANAGER )
+		.getLiveContainerAnalyticsPropertyID( accountID, internalContainerID );
+
+	const { buildAndReceiveWebAndAMP } = createBuildAndReceivers( registry );
+	buildAndReceiveWebAndAMP( {
+		accountID,
+		webPropertyID: propertyID,
+		ampPropertyID: propertyID,
+	} );
+}
+
+function provideWebContainer( registry, accountID ) {
```

Comment:
I'd expect a `provide` helper to receive the container itself as well. This is essentially just selecting the first web container that is available for the given account. This is fine as a utility, I'd just name it differently to make that a bit more clear. Perhaps something like `selectFirst(Web|AMP)Container`?

Reply:
Updated ✔️